assets/img/icons/link-external-white.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why not use the link-external.svg we already have and just change `fill="#1a1a1a"` to `fill="#FFFFFF"?

Reply:
Following on from your comment above:

The problem with having SVGs as <img> tags is that we are very limited with what we can do with them. For example, if we wanted to change the color, or even make it larger, we would have to create a new SVG and change it in the source code. That's why I believe it would be better to be using either inline SVG's or use something like Fontawesome for all our icons.